### PR TITLE
PR: Add links for OpenTeams

### PR DESCRIPTION
**Pull request by dalthviz**
# Pull Request

## Pull Request Checklist

* [x] Read and followed this repo's [Contributing Guidelines](https://github.com/spyder-ide/spyder/blob/master/CONTRIBUTING.md)
* [x] Based your PR on the latest version of the correct branch (master or 4.x)
* [x] Checked your writing carefully for correct English spelling, grammar, etc
* [x] Described your changes and the motivation for them below

## Description of Changes

Some links requested from OpenTeams to inform users that there is an option to request support through that channel.

### CODE REVIEW

File: doc/_static/css/custom_styles.css [hunk 1]
```diff
@@ -21,6 +21,11 @@ strong {
 
 /* Style for Font Awesome that only styles the icons before elements */
 
+.openteams-icon::before {
+  content: url("../images/Opeteams_Favicon-01.svg");
+  width: 1em;
+}
+
```
Comment (CAM-Gerlach) [inline doc/_static/css/custom_styles.css:28]:
```suggestion

/* Style for OpenTeams icon, used like the FA Brands icons */

.openteams-icon::before {
  content: url("../images/openteams-favicon.svg");
  width: 1.3em;
  margin-left: -0.2em;
}

div.openteams-icon::before {
  margin-right: 0;
  padding-bottom: 0;
}

```

This fixes the faux-FAB icon's size and alignment not matching the others, fixes the file name and adds an appropriate descriptive comment. Aside from these changes, you'll need to move this to the bottom of the FA icons section (just above the first `iframe` rule), as it is a bespoke override to them (so the styles apply correctly, as well as it makes more sense).

File: doc/installation.rst [hunk 2]
```diff
@@ -349,9 +349,14 @@ Additional help
 
 *Asking a quick question or want to chat with the dev team?* Stop by our `Gitter chatroom`_.
 
+.. rst-class:: fasb openteams-icon
+
+Get help from expert Spyder consultants on `OpenTeams`_.
```
Comment (CAM-Gerlach) [inline doc/installation.rst:354]:
```suggestion
*Need personalized help from expert Spyder consultants?* Visit `OpenTeams`_.
```

Ensure this matches the structure of all the others, `*<Question>?* <Action> <Link>`

File: doc/troubleshooting/call-for-help.rst [hunk 3]
```diff
@@ -72,6 +72,14 @@ It has an vibrant Spyder community as well, with new questions posted every day,
 
 .. _Stack Overflow: https://stackoverflow.com/questions/tagged/spyder
 
+.. rst-class:: fabb openteams-icon
```
Comment (CAM-Gerlach) [inline doc/troubleshooting/call-for-help.rst:75]:
```suggestion
.. _Stack Overflow: https://stackoverflow.com/questions/tagged/spyder


.. rst-class:: fabb openteams-icon
```

Missing line break

**Review by steff456 — APPROVED** [review]
This looks good for me! Thanks for working in this so fast @dalthviz :)

File: doc/troubleshooting/call-for-help.rst [hunk 4]
```diff
@@ -72,6 +72,14 @@ It has an vibrant Spyder community as well, with new questions posted every day,
...
+
+OpenTeams support
+~~~~~~~~~~~~~~~~~
+
+Get help from expert Spyder consultants on `OpenTeams`_. The OpenTeams marketplace is your one source for everything open source.
```
Comment (CAM-Gerlach) [inline doc/troubleshooting/call-for-help.rst:80]:
```suggestion
On `OpenTeams`_, you can get help from expert Spyder consultants.
OpenTeams offers services ranging from a brief consultation to an experienced team to help you with a large scale implementation project using Spyder.
```

Conform to OSPL, match the structure of the others, give more useful and specific details about what services they provide for Spyder users (using the copy straight off their linked site) and avoid meaningless advertising taglines that don't communicate any actual information to the user (per the [Style Guide](https://github.com/spyder-ide/spyder-docs/blob/master/STYLEGUIDE.md#writing-all-languages),

> Focus on the user; this is documentation, not promotional material.

File: doc/_static/images/Opeteams_Favicon-01.svg [hunk 5]
```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (CAM-Gerlach) [inline doc/_static/images/Opeteams_Favicon-01.svg:1]: I can't make it as a suggestion, but could you rename this to `openteams-favicon.svg`? It fixes a typo in the filename, follows the conventions used for the rest of the images (the naming of `spyder_logo`/`_banner` was an oversight when Isabella updated it), and avoids a superfluous suffix.

**Review by CAM-Gerlach — CHANGES_REQUESTED** [review]
These review suggestions fix some rendering issues, refine the added text to match the structure, tone and focus of the others and add a missing line break. I also requested a rename on the image to be consistent and correct with the others here.

As a sidenote, I noticed a few issues on the OpenTeams page, if you can do it or let them know about it:

* The proper name of the project is "Spyder", not "spyder".
* There are spurious `\n` (line break) characters in the project description
* If you can, you should have to have them track the CF version; it seems they are tracking the very out of date Anaconda version (as 5.0.5 is the latest version listed there, which is stated to be many months old).

File: doc/troubleshooting/call-for-help.rst (hunk 4 above)
Comment (steff456) [reply doc/troubleshooting/call-for-help.rst:80]: I'm unsure about the second part suggested, I don't know if they offer services "ranging from a brief consultation to an experienced team". And I also want to note that the marketing team of open teams send us the texts they needed.

Comment (CAM-Gerlach) [reply doc/troubleshooting/call-for-help.rst:80]: Actually, as implied above, this is their own official copy (warts and all), as its an exact quote of what they say near the top-left of their page linked in the previous sentence:

>  Many spyder users find that they occasionally need help with installation or debugging a tricky application. **OpenTeams offers services from spyder experts ranging from a brief consultation to an experienced team to help you with a large scale implementation project.**

Including a sponsor plug is okay, but it needs to fit in an appropriate context to be effective and ethical. A pure advertising slogan like "the OpenTeams marketplace is your one source for everything open source" is fine in a dedicated section plugging the sponsor (perhaps even on a more prominent page like the index), but doesn't really make sense embedded in a section of the troubleshooting documentation listing help resources for Spyder and explaining what sort of assistance they can get at each one, whereas an except from their Snyder-specific description listing the services they provide does. This both actually gives readers useful information about what OpenTeams can offer them for troubleshooting, while also making a more compelling pitch for using their services tailored to the specifics of the context.

Comment (CAM-Gerlach) [reply doc/troubleshooting/call-for-help.rst]: Great, glad to hear that—thanks, @dalthviz

File: doc/installation.rst [hunk 6]
```diff
@@ -349,9 +349,14 @@ Additional help
 
 *Asking a quick question or want to chat with the dev team?* Stop by our `Gitter chatroom`_.
 
+.. rst-class:: fasb openteams-icon
+
+*Need personalized help from expert Spyder consultants?* Visit `OpenTeams`_.
```
Comment (CAM-Gerlach) [inline doc/installation.rst:354]:
```suggestion
*Seeking personalized help from expert Spyder consultants?* Visit `OpenTeams`_.
```

Sorry, this was my mistake—need is repeated from a few lines above, when every other line has its own beginning word. I fixed it in my PR against your branch as well, so you don't need to apply this if you accept that.

**Review by CAM-Gerlach — COMMENTED** [review]
Since I couldn't make the style move fix as a suggestion, I made that and the other word fix as a PR, dalthviz/spyder-docs#1 , against your branch.

File: doc/installation.rst [hunk 7]
```diff
@@ -349,9 +349,14 @@ Additional help
 
 *Asking a quick question or want to chat with the dev team?* Stop by our `Gitter chatroom`_.
 
+.. rst-class:: fasb openteams-icon
```
Comment (CAM-Gerlach) [inline doc/installation.rst:352]:
```suggestion
.. rst-class:: fabb openteams-icon
```

**Review by CAM-Gerlach — COMMENTED** [review]
One remaining issue with the styles not applying due to the class being applied here, that I didn't catch in testing due to the inheritance not being quite the same. Since its such a tiny change, I'll apply it and then merge this so as to not delay this further.

**Review by CAM-Gerlach — APPROVED** [review]
LGTM, thanks for bearing with us @dalthviz !

**Pull request merged**